tssc/__init__.py

```diff
@@ -12,8 +12,9 @@
   -e ENVIRONMENT, --environment  ENVIRONMENT
         The environment to run this step against.
 
-  -c CONFIG_FILE, --config-file CONFIG_FILE
-        TSSC workflow configuration file in yml or json
+  -c CONFIG_FILE [CONFIG_FILE ...], --config-file CONFIG_FILE [CONFIG_FILE ...],
+  --config-files CONFIG_FILE [CONFIG_FILE ...], --config-dir CONFIG_FILE [CONFIG_FILE ...]
```

Comment:
need to update the config-dir text. I also feel like --config-files should be dropped given --config-file already supports the multiple, at least per what you wrote. Is there a way to make -c behave the same as --config-file?

Reply:
they all beheive the same. the point is i thought it was weird to have --config-file take multiple files, but i also didn't want to make different logic. so `-c` `--config-file` `--config-files` and `--config-dir` are all parsed as the same argument and handled the same way. If you all want to standerdize down to one flag, not to late i suppose, pick one.

Reply:
If they all act the same way, it needs to be noted.  `--config-files` seems weird to me, and I would just drop it entirely.

Reply:
Agreed it needs to be noted and the flag should be the lowest common denominator indicating the input type, like `-c` and `--config` where the documentation notes that what can be passed here is a file, files, or dir.  Adding file, files, or dir to it might be too specific for those that don't read the note.

Reply:
Since the code has `nargs='+'` something like `.... -config-files config1.yml config2.yml` looks more reasonable than the other spelled out options. But I'm cool either way, there's alot of aliases there :). Also the comma separation implies the options are all equivalent but a comment confirming that definitely wouldn't  hurt things

Reply:
ask and you shall  recieve. it is now reduced down to `-c` and `--config`.